### DISCUSSION TITLE
ScrollView behind statusbar, but content starts slightly scrolled so its not behind statusbar

### DISCUSSION THREAD

**Discussion by Noitidart** [software-mansion/react-native-screens#856] [Q&A] [answered]
This is iOS only. I am on iOS 10.

I posted this question here - https://stackoverflow.com/questions/66666279/prvent-scrollview-positioned-at-top-from-mounting-content-slightly-scrolled-to

I created a snack here - https://snack.expo.io/@noitidart/native-navigation-scrollview-content

Ill repeat the question from stackoverflow:

My code for the screen is this:

```
<ScrollView
  style={{ backgroundColor: 'midnightblue' }}
  contentContainerStyle={{ backgroundColor: 'pink' }}
>
   <Text>Hi</Text>
</ScrollView>
```

On mount, the scrollview starts at the top of the screen, below the status bar, as I expect. However the content is slightly scrolled so the content is not behind the statusbar. I want the content behind the statusbar though. Is there a way to prevent this?

![](https://i.stack.imgur.com/x4L5E.png)

**Comment by kacperkapusciak** [comment, marked as answer]
Hi @Noitidart,
I've run your example on iOS 14 and it works fine.
<details>
<summary>video</summary>
<br>

https://user-images.githubusercontent.com/39658211/111448623-e3701f00-870e-11eb-9099-d5a32d1fd3a6.mov

</details>


Your issue probably comes from this code in core react-native 
https://github.com/facebook/react-native/blob/master/React/Views/ScrollView/RCTScrollView.m#L290
and there's really nothing we can do about it to make it work <iOS 11.

Cheers

**Reply by Noitidart** [reply]
Thanks very much @kacperkapusciak - my issue is on iOS 10, I haven't been able to test on anything later then yet. 

I way appreciate your fast reply. Do you mean >=iOS11 it should be no problem, but <iOS11 we're low on options? Sorry just clarifying.

**Reply by Noitidart** [reply]
I tried setting `automaticallyAdjustContentInsets` https://reactnative.dev/docs/scrollview#automaticallyadjustcontentinsets-ios to false but no luck :(

**Reply by kacperkapusciak** [reply]
Yes, sorry I've made a typo with `<` sign. I've edited my answer. It should work just fine for versions iOS 11+. I've tested iOS 12, 13, 14 and it works. I am not able to test older versions.

**Reply by Noitidart** [reply]
Thank you very much! I will keep trying and if I come up with anything will share. If i don't i will come saying so and close. :)
